### ISSUE TITLE
how to generate test masks?

### ISSUE THREAD

**Issue by qsc227** [nipponjo/deepfillv2-pytorch#13]
Hello,Thanks for your code! I would like to know how do you  generate your test masks? I find that when i try to use pretrained models to test on my masked images,the results seems to be not very good.
<img width="319" alt="image" src="https://user-images.githubusercontent.com/70196171/218672831-13d74dd2-d189-403f-9be3-63398a244fb3.png">
<img width="320" alt="image" src="https://user-images.githubusercontent.com/70196171/218673073-bb557464-1823-4b50-9d82-2ee576928bcc.png">

**Comment by nipponjo** [comment]
Hello, I used the demo app to generate the masks in the examples/inpaint/ folder.

**Comment by qsc227** [comment]
why i try to push the impaint button , there is nothing like this?
<img width="870" alt="image" src="https://user-images.githubusercontent.com/70196171/218681568-9403ff4b-b8f3-4944-91da-a49b5857c209.png">

**Comment by juntaosun** [comment]
我也遇到同样的问题，使用示例的图片和mask一切正常
I also have the same problem, Everything works fine with the picture and mask of the example

当我使用photoshop进行蒙版绘制，它的结果不正常。在PS中使用白色画笔和透明区，保存为png格式。
when I use photoshop for mask drawing, it doesn't work. Use white brush and transparent area in PS, save as png format.

那么在PS中，该如何正确绘制mask?
So in PS, how to draw the mask correctly ?

**Comment by nipponjo** [comment]
It seem like PS saves the channels in a different order, so that
`mask = mask[0:1, :h//grid*grid, :w//grid*grid].unsqueeze(0)` (in `test.py`)
should be changed to
`mask = mask[3:4, :h//grid*grid, :w//grid*grid].unsqueeze(0)`

I will look into this further later.
